docs/Learning-Environment-Examples.md

```diff
@@ -36,7 +36,7 @@ you would like to contribute environments, please see our
   - +1.0 for arriving at optimal state.
 - Behavior Parameters:
   - Vector Observation space: One variable corresponding to current state.
-  - Vector Action space: (Discrete) Two possible actions (Move left, move
+  - Vector Action space: 1 discrete action branch with 3 actions (Move left, do nothing, move
```

Comment:
Vector Action Space

Reply:
Will rename all of these based on discussion of UI changes.